app/scripts/templates/sign_in.mustache

```diff
@@ -19,9 +19,13 @@
     {{^error}}
       <div class="error"></div>
       <div class="success"></div>
-        {{#isMigration}}
+      {{#isSyncMigration}}
           <div class="info nudge">{{#t}}Migrate your sync data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
-        {{/isMigration}}
+      {{/isSyncMigration}}
+
+      {{#isAmoMigration}}
+          <div class="info nudge">{{#t}}Migrate your Mozilla Add-ons data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
```

Comment:
This string isn't in @ryanfeeley's migration flow for AMO, which only has migration-specific text on the sign-up page.  The simplest first step may be to keep the sync-specific messaging, but don't show anything for other migrations (which is what I think @shane-tomlinson is suggesting in https://github.com/mozilla/fxa-content-server/issues/3399#issuecomment-180338523)